.github/scripts/release_notes/constants.js

```diff
@@ -75,7 +76,8 @@ const EXTENSION_ENUM = Object.freeze({
     AUDIENCE: "Audience",
     CAMPAIGN_STANDARD: "Campaign Standard",
     CAMPAIGN_CLASSIC: "Campaign Classic",
-    NOTIFICATION_CONTENT: "Notification Content Extension"
+    NOTIFICATION_CONTENT: "Notification Content Extension",
+    UI: "UI"
```

Comment:
Can we use a more specific name for this Android extension? This extension name is used as part of the headings of the release note items, for example, [iOS Notification Content Extension 5.0.0](https://developer.adobe.com/client-sdks/home/release-notes/#ios-notification-content-extension-500).  `Android UI 3.0.0` looks a bit too generic. Can we rename it?
cc @sbenedicadb @praveek

Reply:
ATM the UI library has only Notification Builder module. But in the future it might have more UI templates which is why I left it generic. How about "UI Library"? Happy to hear suggestions

Reply:
This repo will contain  multiple artifacts similar to Core. What is the artifact name and tag prefix for push template utils? Can you update the PR to use those instead of UI?

Reply:
Updated to "Notification Builder"